examples/big_bank.py

```diff
@@ -27,7 +28,7 @@
 from structurizr.view import ElementStyle, PaperSize, RelationshipStyle, Shape
 
 
-def main():
+def create_big_bank():
```

Comment:
Please don't rename this function. I rely on the existence of a `main` function in every example to create the workspace in some other code. (Should be documented, of course.)

You're welcome to create other functions, though, called by `main` that construct a specific part of the example, as long as `main` returns the workspace.

Reply:
Done.